OrbitGl/PickingManager.h

```diff
@@ -101,22 +97,24 @@ class PickingManager {
   void Pick(uint32_t a_Id, int a_X, int a_Y);
   void Release();
   void Drag(int a_X, int a_Y);
-  Pickable* GetPicked();
-  Pickable* GetPickableFromId(uint32_t id);
-  bool IsDragging() const;
+  std::weak_ptr<Pickable> GetPicked();
+  std::weak_ptr<Pickable> GetPickableFromId(uint32_t id);
+  bool IsDragging();
```

Comment:
I went throught the different implementations of IsDragging and I don't see why the const needs to go away. Can you briefly explain?

Reply:
I was under the impression `lock` wasn't `const`, again because of a compiler error that indicated so... I think at some point there was something else wrong in the code which lead me to change this and the "virtual" from your comment above, that's weird. I'll add it back.